Lib/locale.py

```diff
@@ -262,7 +266,7 @@ def currency(val, symbol=True, grouping=False, international=False):
         raise ValueError("Currency formatting is not possible using "
                          "the 'C' locale.")
 
-    s = format('%%.%if' % digits, abs(val), grouping, monetary=True)
+    s = format_string('%%.%if' % digits, abs(val), grouping, monetary=True)
```

Comment:
Since we *know* there's only one format string here, I think this should be a call to _format.